_sass/_components/sticky.scss

```diff
@@ -2,6 +2,7 @@
   position: -webkit-sticky;
   position: sticky;
   top: 0;
+  padding-right: $paragraph-margins;
```

Comment:
This is what we have now:

<img width="541" alt="screen shot 2017-02-09 at 5 58 55 pm" src="https://cloud.githubusercontent.com/assets/4803473/22811061/7384341a-eef1-11e6-9f9b-cbb909fc8faf.png">